libvast/builtins/formats/json.cpp

```diff
@@ -186,13 +249,14 @@ class plugin final : public virtual parser_plugin,
     };
     return [](generator<chunk_ptr> json_chunk_generator,
               operator_control_plane& ctrl) -> generator<table_slice> {
+      // TODO: change max table slice size to be fetched from options.
+      constexpr auto max_table_slice_rows = defaults::import::table_slice_size;
       auto parser = simdjson::ondemand::parser{};
       auto stream = simdjson::ondemand::document_stream{};
       auto slice_builder = adaptive_table_slice_builder{};
-      // The simdjson suggests to initialize the padding part to either 0s or
-      // spaces.
-      auto json_to_parse_buffer
-        = detail::padded_buffer<simdjson::SIMDJSON_PADDING, '\0'>{};
+      auto json_to_parse_buffer = json_buffer{};
+      auto line_parser
```

Comment:
There is a comment when we use this object. I'd rather not have the same comment twice (on instantiation and on usage). I think the explanation makes sense when it is used.

Reply:
Can you perhaps move them closer together, e.g., by moving this object to the comment rather than duplicating the comment? I don't understand why it would need to be this far up.

Reply:
It creates the simdjson::parser once so that we can reuse it's internal buffers for parsing. I can move it as the performance in this rare path shouldn't be critical. Any thoughts? Move it or leave it?

Reply:
I decided to reuse the main parser and move the instantiation close to the usage.